lib/iiif/presentation/annotation.rb

```diff
@@ -7,8 +7,11 @@ class Annotation < AbstractResource
       TYPE = 'oa:Annotation'
 
       def required_keys
-        super + %w{ motivation }
+        super + %w{ motivation on }
       end
+      
+      def string_only_keys
+        super + %w{ on }
 
```

Comment:
Looks like you're missing an `end` for this method.  You aren't by chance a Python programmer are you 😉 ?

Reply:
Ooops! 

I'm actually more of a Perl programmer, but I was working on Python earlier in the day, so maybe that was it!

Actually, looking at my local copy, I've got the "end" in there. I didn't copy and paste so my head must have still been in Python mode ;)

Reply:
Added another commit to fix that missing "end".

I'm using the Github browser editor out of laziness/a desire for speed, but I admit I'm not so familiar with the web interface. Used to doing everything on the CLI...